src/tests/test_alldebrid_downloader.py

```diff
@@ -0,0 +1,162 @@
+import json
+
+import pytest
+from program.downloaders import alldebrid
+from program.downloaders.alldebrid import (
+    AllDebridDownloader,
+    add_torrent,
+    get_instant_availability,
+    get_status,
+    get_torrents,
+)
+from program.settings.manager import settings_manager as settings
+
+
+@pytest.fixture
+def downloader(instant, upload, status, status_all):
+    """Instance of AllDebridDownloader with API calls mocked"""
+    # mock API calls
+    _get = alldebrid.get
```

Comment:
Poor man's mocking. It didn't seem worth bringing in a mocking lib for this one mock

Reply:
We are going to have to mock other api calls elsewhere anyways as the whole app needs to have testing.. but it changes so frequently that its hard to stay on top of. Appreciate the unit test!